### PR TITLE
Popup

### PR DESCRIPTION
**Pull request by Elijahdre**
## Pop up menu
- I fixed the pop up to display more details about the project for both mobile and desktop.
- I use the correct work flow
- I ran linters commands.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -580,6 +590,69 @@ textarea {
...
+.popup_container {
+  z-index: 10000;
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  background-color: rgba(124, 124, 124, 0.7);
+  align-items: center;
+  justify-content: center;
+  position: fixed;
+}
```
Comment (mwanawabangona) [inline style.css:602]: - [ ] `required`
- [ ] Amazing job so far! :clap: 
- [ ] I suggest that you make some adjustments to the `pop-up cards`, please make them as much as similar to the figma template, kindly follow the `background blur`, `fonts` and how the `background images` are placed please see screenshots below
- `figma template`
Screenshot from 2022-07-21 20-00-11

- `your template`
Screenshot from 2022-07-21 20-03-09

File: style.css [hunk 2]
```diff
@@ -580,6 +590,69 @@ textarea {
...
+  border-radius: 25px;
+  width: 75%;
+  padding: 1rem;
+}
+
+.popup_head {
+  display: flex;
+  justify-content: space-between;
+}
+
```
Comment (mwanawabangona) [inline style.css:615]: - [ ] `required`
- [ ] as mentioned above in the desktop `pop-up`
- [ ] I suggest that you make some adjustments to the `pop-up cards`, please make them as much as similar to the figma template, kindly follow the background blur, fonts and how the background images are placed please see screenshots below

- `your template`
Screenshot from 2022-07-21 20-11-34

- `figma template`
Screenshot from 2022-07-21 19-59-38

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @Elijahdre,

Amazing  job so far!

There are some issues that you still need to work on :pinching_hand:  to go to the next project but you are almost there!

## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Elijahdre) [reply style.css:602]: @mwanawabangona OK I will stick to the design better and fix it. Thank you very much

File: style.css (hunk 2 above)
Comment (Elijahdre) [reply style.css:615]: @mwanawabangona Thank you again.

**Pull request merged**